infra/versions.tf

```diff
@@ -19,7 +19,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 4.75.0"
+      version = "< 4.84.1"
```

Comment:
This change is dropping the >= 3.53 constraint, is that alright?

@pattishin @rogerthatdev do you have insight on this?

Reply:
This appears to be an instance of https://github.com/renovatebot/renovate/issues/17973